docs/pages/database-access/reference/configuration.mdx

```diff
@@ -54,6 +53,9 @@ proxy_service:
   mongo_public_addr: "mongo.teleport.example.com:443"
 ```
 
+Database types that support native TLS (Redis, ElasticSearch, OpenSearch) share
+the same port specified by `proxy_service.public_addr`.
```

Comment:
This is not totally true for instance MySQL connection can be also established to public_addr via TLS Routing that is currently the default behaviour. Additionally MSSQL protocol doesn't support top TLS teleport the TLS handshake is done after some TDS MSSQL client/server hello handshake. 

the `tsh db connect` and `tsh proxy db`  commands hides this complexity and the end user should bother with port should select for what db protocol so I want suggest  to just remove this line.